build.gradle.kts

```diff
@@ -17,13 +17,13 @@ dependencyCheck {
 
 dependencies {
   implementation("com.beust:klaxon:5.6")
-  implementation("com.amazonaws:aws-java-sdk-s3:1.12.266")
-  implementation("io.sentry:sentry-spring-boot-starter:6.1.4")
-  implementation("net.javacrumbs.shedlock:shedlock-spring:4.37.0")
-  implementation("net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.37.0")
+  implementation("com.amazonaws:aws-java-sdk-s3:1.12.279")
+  implementation("io.sentry:sentry-spring-boot-starter:6.3.1")
+  implementation("net.javacrumbs.shedlock:shedlock-spring:4.39.0")
+  implementation("net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.39.0")
   implementation("nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect:3.1.0")
   implementation("org.apache.poi:poi-ooxml:5.2.2")
-  implementation("org.flywaydb:flyway-core:8.5.13")
+  implementation("org.flywaydb:flyway-core")
```

Comment:
Don't need to specify a version here.  This is managed/handled by Spring boot automatically.